src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,38 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
```

Comment:
Consider alphabetizing the import statements for better readability and consistency. 📚